Unit tests fail depending on build environment locale

#### Here's what I did
Tried to build current dev code on a machine with default system locale de-DE 

---
#### Here's what I got
```
[ERROR]   IsochronesRequestHandlerTest.processIsochronesRequestOptionsTest:322 expected:<0.5> but was:<-1.0>
[ERROR]   RouteRequestHandlerTest.convertRouteRequestTest:211 expected:<0.5> but was:<-1.0>
[ERROR]   RouteRequestHandlerTest.convertRouteRequestTest:181 expected:<0.5> but was:<-1.0>
```

---
#### Here's what I was expecting
a successful build

---
#### Here's what I think could be improved

The affected tests check `ProfileWeighting` objects that use `com.graphhopper.util.PMap` to store double values, which internally stores everything as string, therefore creating a locale/culture issue. 

- set maven-surefire params to use locale en-US for testing fixes the build process
- we should however investigate if we can avoid using this util class altogether or fix it before we run into more obscure issues.

--- a/openrouteservice/pom.xml
+++ b/openrouteservice/pom.xml
@@ -181,6 +181,13 @@
           <version>2.6</version>
       </plugin>
 
+        <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+                <argLine>-Duser.language=en -Duser.region=US</argLine>
+            </configuration>
+        </plugin>
       </plugins>
   </build>
 

